docs/use-cases.md

```diff
@@ -0,0 +1,19 @@
+# Key Transparency Use Cases
+
+
+Key Transparency ensures, with mathematical certainty, that account owners can
+see all the public keys that have been associated with their account.  
```

Comment:
Would it be worth elaborating a bit on what "associated" means? Like, some other user using my key is included here, right?

Reply:
How's "see all the public keys that have been used to authenticate their account or send them messages." ?